### DISCUSSION TITLE
【灌水】ZZZ和原神的随手拍

### DISCUSSION THREAD

**Discussion by DZY-QAQ** [share121/inter-knot#1152] [其他]
**ZZZ照片**
![20240810212334](https://github.com/user-attachments/assets/304be6f3-4532-400a-880d-8cdae07d846e)
![20240810212437](https://github.com/user-attachments/assets/5affb4b7-e253-4924-96f7-383026869202)
![20240810212624](https://github.com/user-attachments/assets/80b55063-6ec8-4ab9-8654-92f17414c757)
**原神照片**
![20240805151927](https://github.com/user-attachments/assets/ea38342a-b03a-46f0-b19e-cb603c89e26f)
![20240803154618](https://github.com/user-attachments/assets/f4a5695f-d476-4ce0-b7fe-a8347c5e14ae)
![20240803154911](https://github.com/user-attachments/assets/85c5b393-cd8c-4037-bc00-7496bd75eaf8)
![20240801105104](https://github.com/user-attachments/assets/043e7c98-35fe-402f-bf79-28a91e64bb4d)

**Comment by 1451374815** [comment]
好看

**Comment by Hongxuli2006** [comment]
这不是阿原拍照活动吗
